cmd/wcloud/cli.go

```diff
@@ -29,10 +29,10 @@ var (
 
 func usage() {
 	fmt.Println(`Usage:
-	deploy <image>:<version>   Deploy image to your configured env
-	list                       List recent deployments
-	config (<filename>)        Get (or set) the configured env
-	logs <deploy>              Show lots for the given deployment`)
+	deploy (-m) <image>:<version>   Deploy image to your configured env
```

Comment:
Ah, I see it is *not* set in the config. I sort of expected it to be, similar to how Slack notifications are set.

Setting it on deploy is probably fine for now, but should document `(-m "commit_message")`